code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_abilities.dm

```diff
@@ -21,6 +21,8 @@
 	if (X.mutation_type == LURKER_NORMAL)
 		var/found = FALSE
 		for (var/mob/living/carbon/human/H in get_turf(X))
+			if(H.stat == DEAD)
```

Comment:
Please also fix the single letter vars here. See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#variables
You can use VSC to do this easily by selecting the function, search, enable dropdown for replace, enable word, enable case, and then for H/X and replace with `human`/`xeno` or something.